amica/src/lib/vad/real-time-vad.ts

```
@@ -0,0 +1,716 @@
+
+        log.info('[VAD] AudioWorkletNode created successfully')
+
+        ;(this.audioNode as AudioWorkletNode).port.onmessage = async (
+          ev: MessageEvent
+        ) => {
+          switch (ev.data?.message) {
+            case 'WORKLET_INITIALIZED':
+              log.debug('[VAD] Worklet initialized!', ev.data);
+              break
+            case Message.AudioFrame:
+              let buffer: ArrayBuffer = ev.data.data
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)